server/pecl/settings.py

```diff
@@ -265,15 +265,8 @@
     "config": {"tools": [{}], "agent_args": {}, "prompt_input": {}, "prompt_extension": {}},
 }
 
-FILE_PARSER_CLASSES: dict[tuple[str] : str] = {
-    (".txt",): "enthusiast_file_parsers.text.PlainTextFileParser",
-    (".pdf",): "enthusiast_file_parsers.pdf.PDFFileParser",
-    (
-        ".jpg",
-        ".jpeg",
-        ".png",
-    ): "enthusiast_file_parsers.image.ImageFileParser",
-}
+FILE_PARSER_CLASSES: dict[tuple[str] : str] = {}
```

Comment:
Don't they need to be configured by default?

Reply:
I'll leave it empty as there are no any parser implementation on server and enthusiast-file-parser is not installed. We will add it to enthusiast-starter along with file-parser-package.

Reply:
Does it really have to be so hard to use?

Reply:
What's preventing us from having these parsers as a part of the server, and having them configured by default?